### ISSUE TITLE
Copy Enterprise site to Enterprise site succeeds but sites not accessible

### ISSUE THREAD

**Issue by niwacolours** [Esri/hub-py#92]
I've used a somewhat modified example from https://github.com/Esri/hub-py/blob/master/examples/For arcgishub/Working with Sites-Hub.ipynb to copy my existing sites from a Pre-Production ArcGIS Enterprise 10.7.1 portal to our Production equivalent.
(The Esri example oddly doesn't have a case where a clone runs from one Enterprise Portal to another)

![image](https://user-images.githubusercontent.com/53462390/77598601-274e9f00-6f67-11ea-8930-6a6deeecfcbe.png)

Copying seems to work (as I can see the sites are created in the target portal, even though they have a XYDid_copy appended to their name which I can't change) but accessing/using them is a little tricky - I can use one of the target sites when I first open it in a browser but when I use the navigation inside it which links to the other sites results in an error "Access Denied" for all those other copied sites:
![image](https://user-images.githubusercontent.com/53462390/77598780-93310780-6f67-11ea-948c-575ccf36e9d9.png)

which is more odd after I've shared them all publicly: 
![image](https://user-images.githubusercontent.com/53462390/77598826-b956a780-6f67-11ea-81ee-191bf957241e.png)

I've tried to make another user the owner and use those credentials - to no avail. 

Also the "Configuration" tab for all the copied Sites is empty - whereas "Groups/Capabilities/Pages/ and the Site Editor" are functioning correctly.

For what it's worth: we are using Enterprise logins in both portals but in the script, I've used an ArcGIS Builtin User to run the copy process. 

Any ideas/hints very much appreciated.

**Comment by ManushiM** [comment]
@niwacolours thanks for bringing this to my attention! The first thing I notice is that you mention cloning from an ArcGIS Enterprise 10.7.1 portal, however per the README https://github.com/Esri/hub-py/blob/master/README.md, this API supports sites in Enterprise versions 10.8 and above.

Also, can you please elaborate on what you mean by `ArcGIS Builtin User`?

**Comment by niwacolours** [comment]
Hi Manushi, thanks for getting back to me – really appreciate it, seeing how difficult it is for everyone at the moment to keep up with work and life!

I had actually not realized that this was from version 10.8 only – and can report that (at least in part) it works for 10.7.1 (!)

I’ve also been able to resolve my difficulties with the various sites not being available – a few things were involved in fixing that (and I can only blame myself for not checking them before posting, sorry!):


  1.  Sites were renamed during copy and, using the Site Administration page (NOT the item details page) I have been able to re-rename them so they have their original name
  2.  What errors I got were related to the URLs to access sites using a custom navigation (that I have in the header) – those URLs were obviously the same as in the source portal as they had been hardcoded and weren’t pointing to the correct targets after the copying (and before I had managed to re-rename them to their correct names). The error message was misleading be.
  3.  The fact that we are using SAML based users didn’t have anything to do with it, I now think. What I meant by ArcGIS Built-in users are “Built-in Portal members” (see screenshot below): [cid:image002.png@01D61330.662D96B0]


So I think it’s all working now!

Another thing I’ve been trying since was to copy pages – from one portal (Pre-Prod) to another (Prod) – with no success. I think the error was about a “Collaboration property not set” or something similar. Should (part of) that work for 10.7.1?


Thanks again – cheers,

Tilmann
